WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -34,35 +43,35 @@ function DelegatesOfRegion({ activeSeniorDelegate, delegates, isAdminMode }) {
       <Table.Header>
         <Table.Row>
           <Table.HeaderCell collapsing />
-          <Table.HeaderCell>{I18n.t('delegates_page.table.name')}</Table.HeaderCell>
-          <Table.HeaderCell>{I18n.t('delegates_page.table.role')}</Table.HeaderCell>
-          <Table.HeaderCell>{I18n.t('delegates_page.table.region')}</Table.HeaderCell>
+          <Table.HeaderCell>
+            {I18n.t('delegates_page.table.name')}
+          </Table.HeaderCell>
+          <Table.HeaderCell>
+            {I18n.t('delegates_page.table.role')}
+          </Table.HeaderCell>
+          <Table.HeaderCell>
+            {I18n.t('delegates_page.table.region')}
+          </Table.HeaderCell>
         </Table.Row>
       </Table.Header>
 
       <Table.Body>
         {sortedDelegates([
           activeSeniorDelegate,
-          ...delegates.filter((user) => (
-            user.senior_delegate_id === activeSeniorDelegate.id
-              && user.delegate_status !== 'trainee_delegate'
-          )),
+          ...delegates.filter(
+            (user) => user.senior_delegate_id === activeSeniorDelegate.id
+              && (user.delegate_status !== 'trainee_delegate' || isAdminMode),
```

Comment:
Am I reading it correctly that you are only allowing admins to edit Trainee Delegates?
If yes, how does that make sense with respect to WCA Motions?

Reply:
Actually nope. What I meant was only admins can view trainee delegates. Public users won't be able to see trainee delegates in delegates page because that was what decided by Board (https://github.com/thewca/worldcubeassociation.org/issues/5118).

Note: By admin, I meant the people who can access the delegate admin mode, which include delegates as well. Just pointing it not to get confused with the user.admin of rails, which is WST seniors.

Reply:
Okay, makes sense now!